src/main/java/software/amazon/qldb/exceptions/ExecuteException.java

```diff
@@ -42,18 +42,18 @@ public RuntimeException getCause() {
     }
 
     public boolean isRetryable() {
-        return retryable;
+        return isRetryable;
     }
 
-    public boolean isAbortSuccessful() {
-        return aborted;
+    public boolean isSessionAlive() {
```

Comment:
Will this have the same state as isInvalidSessionException? Do you need to track them seperately?

Reply:
An ISE does not mean that the session is dead. A transaction expiry is classified as an ISE, but the underlying session is still alive.